From 73b621a746e436fb9db771eee240430cc9133953 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Tue, 12 Apr 2022 18:35:43 -0400
Subject: [PATCH] volkpython use posix prefix scheme

Recently in Debian's pthon3.10 the default scheme for sysconfig
changed from 'posix_prefix' to 'posix_local'. VolkPython.cmake
expects the 'posix_prefix' behavior. So explicity use it.

Signed-off-by: A. Maitland Bottoms <bottoms@debian.org>
---
 cmake/Modules/VolkPython.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Modules/VolkPython.cmake b/cmake/Modules/VolkPython.cmake
index 94899b5..12696d0 100644
--- a/cmake/Modules/VolkPython.cmake
+++ b/cmake/Modules/VolkPython.cmake
@@ -121,7 +121,7 @@ try:
 except AttributeError: pass
 if not install_dir:
     #find where to install the python module
-    install_dir = sysconfig.get_path('platlib')
+    install_dir = sysconfig.get_path('platlib','posix_prefix')
     prefix = sysconfig.get_config_var('prefix')
 #strip the prefix to return a relative path
 print(os.path.relpath(install_dir, prefix))"
-- 
2.35.1

